packages/processor/processor.js

```diff
@@ -458,6 +461,10 @@ class Processor {
                 }
 
                 // Otherwise add it to the queue
+                if(this._loadFile) {
+                    return this._loadFile(dependency).then((str) => this.string(dependency, str));
```

Comment:
The call to `this._loadFile` is probably a better fit inside of `Processor.file()`, then it'll be used everywhere instead of just in dependency walking.

Reply:
That was the approach i took originally, but it breaks a lot of tests for some reason. Most breakages are log ordering issues but there was some real things that i didn't quite understand in other pacakges.

Reply:
It's still the right change, being able to set .loadFile and then having the processor ignore it for a bunch of calls will be very weird.

I can definitely help with the test failures!

Reply:
yup, ok i'll do the correct thing and post it up for some input 👍